docs/customizations/footer.md

```diff
@@ -0,0 +1,218 @@
+controlProps | [IFooterControlProps](#ifootercontrolprops) | No | Properties that control the element behariors | -
+styleProps | [IFooterStyleProps](#ifooterstyleprops) | No | Properties that control the element styles | -
+
+### [IFooterComponentOverrides](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterComponentOverrides.ts)
+
+Custom React components can be passed as input to override the default sub-components. Alternatively, you can stringify the React component before passing it in. The `chat-components` library provides one util function that can be used: [encodeComponentString](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/common/encodeComponentString.ts).
+
+| Attribute | Type | Required | Description | Default | 
+| - | - | - | - | - |
+| DownloadTranscriptButton     | ReactNode\|string     | No | Used for overriding default download transcript button | -
+EmailTranscriptButton | ReactNode\|string | No | Used for overriding default email transcript button | -
+AudioNotificationButton | ReactNode\|string | No | Used for overriding default audio toggle button | -
+
+### [IFooterControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/interfaces/IFooterControlProps.ts)
+
+| Attribute | Type | Required | Description | Default | 
+| - | - | - | - | - |
+| id     | string     | No | The top-level element id for the footer | "oc-lcw-footer"
+hideDownloadTranscriptButton | boolean | No | Whether to hide the download transcript button on the footer | false
+hideEmailTranscriptButton | boolean | No | Whether to hide the email transcript button on the footer | false
+hideAudioNotificationButton | boolean | No | Whether to hide the audio notification button on the footer | false
+onDownloadTranscriptClick | () => void | No | The callback function that will be triggered when the download transcript button is clicked | [Starts downloading the whole transcript]
+onEmailTranscriptClick | () => void | No | The callback function that will be triggered when the email transcript button is clicked | [Opens the pane for sending transcript over email]
+onAudioNotificationClick | () => void | No | The callback function that will be triggered when the audio notification button is clicked | [Toggles audio notification on/off]
+downloadTranscriptButtonProps | [ICommandButtonControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/common/interfaces/ICommandButtonControlProps.ts) | No | Properties to further customize the download transcript button | [defaultFooterControlProps](https://github.com/microsoft/omnichannel-chat-widget/blob/main/chat-components/src/components/footer/common/defaultProps/defaultFooterControlProps.ts)
```

Comment:
my suggestion is to reference the link to the specific LoC code block for the setup of this property and the other props , by example -> https://github.com/microsoft/omnichannel-chat-widget/blob/75135c54d4377cc372876e0a57743df3e86d2bd3/chat-components/src/components/footer/common/defaultProps/defaultFooterControlProps.ts#L15C19-L15C19

Reply:
same for emailTranscriptButtonProps and audioNotificationButtonProps

Reply:
Same concern as below. Will not fix for now